src/test/java/roomescape/member/dao/MemberDaoImplTest.java

```diff
@@ -0,0 +1,128 @@
+                        email VARCHAR(255) NOT NULL,
+                        password VARCHAR(255) NOT NULL,
+                        role VARCHAR(255) NOT NULL, 
+                        PRIMARY KEY (id)
+                    );
+                """);
+
+        String insertSql = "INSERT INTO member(name, email, password, role) VALUES (:name, :email, :password, :role)";
+        namedParameterJdbcTemplate.update(
+                insertSql,
+                Map.of(
+                        "name", "admin",
+                        "email", "admin@email.com",
+                        "password", "password123",
+                        "role", "admin"
+                )
+        );
+        namedParameterJdbcTemplate.update(
+                insertSql,
+                Map.of(
+                        "name", "user",
+                        "email", "user@email.com",
+                        "password", "password456",
+                        "role", "user"
+                )
+        );
+    }
+
+    @DisplayName("회원 목록을 조회하는 기능을 구현한다")
+    @Test
+    void findAll() {
+        assertThat(memberDaoImpl.findAll()).hasSize(2);
+    }
+
+    @DisplayName("회원을 아이디로 조회하는 기능을 구현한다")
+    @Test
+    void findById() {
+        Member member = memberDaoImpl.findById(1L).get();
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(member.getId()).isEqualTo(1L);
+        softly.assertThat(member.getName()).isEqualTo("admin");
+        softly.assertThat(member.getEmail()).isEqualTo("admin@email.com");
+        softly.assertAll();
+    }
+
+    @DisplayName("회원을 이메일로 조회하는 기능을 구현한다")
+    @Test
+    void findByEmail() {
+        Member member = memberDaoImpl.findByEmail("user@email.com").get();
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(member.getId()).isEqualTo(2L);
+        softly.assertThat(member.getName()).isEqualTo("user");
+        softly.assertThat(member.getEmail()).isEqualTo("user@email.com");
+        softly.assertAll();
+    }
+
+    @DisplayName("이메일로 회원이 존재하는지 확인하는 기능을 구현한다")
+    @Test
+    void existsByEmail() {
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(memberDaoImpl.existsByEmail("admin@email.com")).isTrue();
+        softly.assertThat(memberDaoImpl.existsByEmail("nonexistent@email.com")).isFalse();
+        softly.assertAll();
+    }
+
+    @DisplayName("새로운 회원을 추가하는 기능을 구현한다")
+    @Test
+    void add() {
+        Member newMember = new Member("new-user", "newuser@email.com", "password789");
+        Member addedMember = memberDaoImpl.add(newMember);
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(addedMember.getId()).isNotNull();
+        softly.assertThat(addedMember.getName()).isEqualTo("new-user");
+        softly.assertThat(addedMember.getEmail()).isEqualTo("newuser@email.com");
+        softly.assertThat(memberDaoImpl.findAll()).hasSize(3);
+        softly.assertAll();
```

Comment:
```suggestion
        SoftAssertions.assertSoftly(softly -> {
            softly.assertThat(addedMember.getId()).isNotNull();
            softly.assertThat(addedMember.getName()).isEqualTo("new-user");
            softly.assertThat(addedMember.getEmail()).isEqualTo("newuser@email.com");
            softly.assertThat(memberDaoImpl.findAll()).hasSize(3);
        });
```

저는 위 방식이 응집도를 높일 수 있다고 보아 좋다고 생각하는데, 리원은 어떻게 생각하시나요??

Reply:
아 네, 아래 방법의 경우 `assertAll()`을 마지막에 작성하지 않아도 되고, 또 해당 테스트 메서드 내에서만 SoftAssertion을 사용하고 있는 만큼 그렇게 쓰는 것이 낫겠네요! 현재로서는 더 이상 사용되지 않는 DaoImpl 관련 코드와 함께 해당 테스트를 제거했는데요, 
향후 테스트 작성 시 이번 피드백을 참고하도록 하겠습니다!